helm/cosmo/charts/router/CHART.md

```diff
@@ -22,6 +22,7 @@ WunderGraph Cosmo router.
 | configuration.otelCollectorUrl | string | `""` | The URL of the Cosmo GraphQL OTEL Collector. Should be internal to the cluster. Default to cloud if not set. |
 | deploymentStrategy | object | `{}` |  |
 | existingConfigmap | string | `""` | The name of the configmap to use for the router configuration. The key "config.yaml" is required. |
+| existingSecret | string | `""` | Existing secret containing the graphApiToken |
```

Comment:
Please document that the secret has to be in the same namespace, named like the chart fullname and the keys has to match with current secret.

Reply:
Sure, I've added a _helper function to set the name. And improved the doc.